File: objC/TestClass.m
#import <Foundation/Foundation.h>

@interface TestClass : NSObject {}

- (void) doSomething;

@end

@implementation TestClass

- (void) doSomething {
	printf("Something \n");
}

@end


int main(int argc, char const *argv[]) {
	TestClass *class = [TestClass alloc];
	[class doSomething];
	return 0;
}

File: C++/TestClass.cpp
#include <iostream>

using namespace std;

class TestClass {
public:
	TestClass() {}
	~TestClass() {}

	void doSomething();
};

void TestClass::doSomething() {
	cout << "Something \n";
}

int main(int argc, char const *argv[]) {
	TestClass t;
	t.doSomething();
	return 0;
}
